observability/network_observability/network-observability-alerts.adoc

```diff
@@ -0,0 +1,43 @@
+
+:_mod-docs-content-type: ASSEMBLY
+[id="network-observability-alerts_{context}"]
+= Network observability alerts
+
+:context: network-observability-alerts
+:toc:
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: Network observability alerts
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+The Network Observability Operator provides a set of alerts for monitoring your {product-title} cluster's network. The alerts are based on the Network Observability Operator's built-in metrics and are designed to give you a quick indication of your cluster's network health.
+//abstract may need to be revised once all the pieces/includes are in place and you can see how the content flows.
+
+
+include::modules/network-observability-alerts-about.adoc[leveloffset=+1]
+
+include::modules/network-observability-configuring-predefined-alerts.adoc[leveloffset=+2]
+
+include::modules/network-observability-alerts-promql-expression.adoc[leveoffset=+2]
+
+include::modules/network-observability-creating-custom-alert-rules.adoc[leveloffset=+1]
+
+include::modules/network-observability-disabling-predefined-alerts.adoc[leveloffest=+1]
+
+////
+
+[id="network-observability-disable-alerts_{context}"]
```

Comment:
should this be removed? (I see the same text written in a different file for disabling alerts)